model/distro/distro_test.go

```diff
@@ -670,3 +670,20 @@ func TestGetDistrosForImage(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Len(t, found, 3)
 }
+
+func TestGetImageIdFromDistro(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	assert.NoError(t, db.ClearCollections(Collection))
+	id := "distro-1"
+	imageID := "distro"
+	d1 := &Distro{
+		Id:      id,
+		ImageID: imageID,
+	}
```

Comment:
You can probably just inline these and use the fields below e.g. `d.ImageId`. Would also recommend using a more realistic example
```suggestion
	d := &Distro{
		Id:      "ubuntu1804-large",
		ImageID: "ubuntu1804",
	}
```

Reply:
Done